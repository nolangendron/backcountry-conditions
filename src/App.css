@import url('https://fonts.googleapis.com/css?family=Raleway&display=swap');
*,
*::before,
*::after {
  box-sizing: border-box;
}

body {
  height: 100%;
  overflow-x: hidden;
  font-family: 'Raleway', sans-serif;
}
