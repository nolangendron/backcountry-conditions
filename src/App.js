import React from 'react';
import './App.css';
import styled from "@emotion/styled";
import size from "../src/styles/size";
import { Header } from "./components/Header";
import Main from "./components/Main";
import NavBar from "./components/NavBar";
import { Footer } from "./components/Footer";
// import { CurrentWeather } from './components/CurrentWeather';

const GridContainer = styled("div")`
display: grid;
height: 100vh;
grid-template-rows: ${size.headerHeight} 1fr ${size.footerHeight};
grid-template-columns: ${size.navBarWidth} 1fr;
grid-template-areas: "header header"
                     "nav main"
                     "footer footer";
`
function App() {
  return (
    <GridContainer>
      <Header />
      <NavBar />
      <Main />
      <Footer />
    </GridContainer>
  );
}

export default App;
