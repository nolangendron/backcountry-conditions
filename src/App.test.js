import React from "react";
import ReactDOM from "react-dom";
import App from "./App";
import { render, cleanup } from "@testing-library/react";

import renderer from "react-test-renderer";

afterEach(cleanup);

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<App />, div);
});

it("matched snapshot", () => {
  const component = renderer.create(<App />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
