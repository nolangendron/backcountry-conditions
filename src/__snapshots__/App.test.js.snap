// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matched snapshot 1`] = `
<div
  className="main-container"
>
  <div
    className="css-2l1hbj"
    selected="powder.webp"
  >
    <div
      className="left-container"
    >
      <div
        className="title"
      >
        <h1>
          <div
            className="css-17jf3v5"
          >
            <span
              aria-label="BC"
              role="heading"
            >
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "0.5s",
                  }
                }
              >
                B
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "0.6s",
                  }
                }
              >
                C
              </span>
            </span>
          </div>
        </h1>
        <h1>
          <div
            className="css-17jf3v5"
          >
            <span
              aria-label="Backcountry"
              role="heading"
            >
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "0.7s",
                  }
                }
              >
                B
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "0.7999999999999999s",
                  }
                }
              >
                a
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "0.8999999999999999s",
                  }
                }
              >
                c
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "1s",
                  }
                }
              >
                k
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "1.1s",
                  }
                }
              >
                c
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "1.2s",
                  }
                }
              >
                o
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "1.2999999999999998s",
                  }
                }
              >
                u
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "1.4s",
                  }
                }
              >
                n
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "1.5s",
                  }
                }
              >
                t
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "1.6s",
                  }
                }
              >
                r
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "1.7s",
                  }
                }
              >
                y
              </span>
            </span>
          </div>
        </h1>
        <h1>
          <div
            className="css-17jf3v5"
          >
            <span
              aria-label="Weather"
              role="heading"
            >
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "1.6s",
                  }
                }
              >
                W
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "1.7000000000000002s",
                  }
                }
              >
                e
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "1.8s",
                  }
                }
              >
                a
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "1.9000000000000001s",
                  }
                }
              >
                t
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "2s",
                  }
                }
              >
                h
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "2.1s",
                  }
                }
              >
                e
              </span>
              <span
                aria-hidden="true"
                className="span"
                style={
                  Object {
                    "animationDelay": "2.2s",
                  }
                }
              >
                r
              </span>
            </span>
          </div>
        </h1>
      </div>
      <div
        className="select"
      >
        <div
          className=" css-2b097c-container"
          onKeyDown={[Function]}
        >
          <div
            className=" css-yk16xz-control"
            onMouseDown={[Function]}
            onTouchEnd={[Function]}
          >
            <div
              className=" css-g1d714-ValueContainer"
            >
              <div
                className=" css-1uccc91-singleValue"
              >
                Select Area...
              </div>
              <div
                className="css-b8ldur-Input"
              >
                <div
                  className=""
                  style={
                    Object {
                      "display": "inline-block",
                    }
                  }
                >
                  <input
                    aria-autocomplete="list"
                    autoCapitalize="none"
                    autoComplete="off"
                    autoCorrect="off"
                    disabled={false}
                    id="react-select-3-input"
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    spellCheck="false"
                    style={
                      Object {
                        "background": 0,
                        "border": 0,
                        "boxSizing": "content-box",
                        "color": "inherit",
                        "fontSize": "inherit",
                        "label": "input",
                        "opacity": 1,
                        "outline": 0,
                        "padding": 0,
                        "width": "1px",
                      }
                    }
                    tabIndex="0"
                    type="text"
                    value=""
                  />
                  <div
                    style={
                      Object {
                        "height": 0,
                        "left": 0,
                        "overflow": "scroll",
                        "position": "absolute",
                        "top": 0,
                        "visibility": "hidden",
                        "whiteSpace": "pre",
                      }
                    }
                  >
                    
                  </div>
                </div>
              </div>
            </div>
            <div
              className=" css-1hb7zxy-IndicatorsContainer"
            >
              <span
                className=" css-1okebmr-indicatorSeparator"
              />
              <div
                aria-hidden="true"
                className=" css-tlfecz-indicatorContainer"
                onMouseDown={[Function]}
                onTouchEnd={[Function]}
              >
                <svg
                  aria-hidden="true"
                  className="css-6q0nyr-Svg"
                  focusable="false"
                  height={20}
                  viewBox="0 0 20 20"
                  width={20}
                >
                  <path
                    d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="bgImage2 none active"
    />
    <div
      className="bgImage2 coquihalla notActive"
    />
    <div
      className="bgImage2 duffey notActive"
    />
    <div
      className="bgImage2 manningPark notActive"
    />
  </div>
</div>
`;
